src/index.html

```diff
@@ -22,6 +22,405 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a
+          class="logo__link"
+          href="index.html"
+        >
+          <img
+            class="logo__img"
+            src="./images/logo.png"
+            alt="logo_MOYO"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="nav__item"
+            data-qa="nav-hover"
```

Comment:
The `data-qa` attribute is used for testing purposes. Ensure that its usage aligns with the testing strategy and that it's necessary here.